From 624c8bd742f47865923e9464f8eeb2cf31e7b4da Mon Sep 17 00:00:00 2001
From: Amit Prakash Ambasta <amit.prakash.ambasta@gmail.com>
Date: Thu, 30 Dec 2021 09:38:37 +0530
Subject: [PATCH] Updated extraver

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 08561a7..6e49eff 100644
--- a/Makefile
+++ b/Makefile
@@ -5 +5 @@ SUBLEVEL = 12
-EXTRAVERSION = -gentoo
+EXTRAVERSION = -ambasta
-- 
2.34.1

