From 3d143f75d25579cdb96ce0451fb4c48e21cb6297 Mon Sep 17 00:00:00 2001
From: Amit Prakash Ambasta <amit.prakash.ambasta@gmail.com>
Date: Wed, 13 Jan 2021 19:12:16 +0530
Subject: [PATCH] Use set mode to select HFR instead of preferred mode

---
 sway/config/output.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/sway/config/output.c b/sway/config/output.c
index b59cabd..f4b56f8 100644
--- a/sway/config/output.c
+++ b/sway/config/output.c
@@ -355,7 +355,9 @@ static void queue_output_config(struct output_config *oc,
 			oc->refresh_rate, oc->custom_mode == 1);
 	} else if (!wl_list_empty(&wlr_output->modes)) {
 		struct wlr_output_mode *mode = wlr_output_preferred_mode(wlr_output);
-		wlr_output_set_mode(wlr_output, mode);
+		set_mode(wlr_output, mode->width, mode->height,
+			mode->refresh_rate, mode->custom_mode==1);
+		// wlr_output_set_mode(wlr_output, mode);
 	}
 
 	if (oc && (oc->subpixel != WL_OUTPUT_SUBPIXEL_UNKNOWN || config->reloading)) {
-- 
2.30.0

